docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+      - media_volume:/vol/web/media
+      - static_volume:/vol/web/static
+    command: >
+      sh -c "python manage.py migrate &&
+            python manage.py wait_for_db &&
```

Comment:
First, you need to make sure that the database is successfully up, and only then proceed with the migrations.